packages/cfd/src/features/Containers/cfd-password-manager-modal.tsx

```diff
@@ -0,0 +1,393 @@
+                        <InvestorPasswordManager
+                            is_submit_success_investor={is_submit_success_investor}
+                            toggleModal={toggleModal}
+                            error_message_investor={error_message_investor}
+                            validatePassword={validatePassword}
+                            onSubmit={onSubmit}
+                            setPasswordType={setPasswordType}
+                            multi_step_ref={multi_step_ref}
+                        />
+                    </Div100vhContainer>
+                </MobileWrapper>
+            </div>
+        </Tabs>
+    );
+};
+
+const CFDPasswordManagerModal = observer(
+    ({
+        is_visible,
+        platform,
+        selected_login,
+        toggleModal,
+        selected_account_type,
+        selected_account_group,
+        selected_server,
+    }: TCFDPasswordManagerModal) => {
+        const { client, ui } = useStore();
+
+        const { email } = client;
+        const { enableApp, disableApp } = ui;
+
+        const { mutate } = useVerifyEmail();
+
+        const sendVerifyEmail = () => mutate({ verify_email: email, type: 'trading_platform_investor_password_reset' });
+
+        const multi_step_ref: React.MutableRefObject<undefined> = React.useRef();
+        const [index, setIndex] = React.useState<number>(0);
+
+        const [password_type, setPasswordType] = React.useState('main');
+
+        if (!selected_login) return null;
+
+        const getTitle = () => {
+            return localize('Manage {{platform}} password', {
+                platform: getCFDPlatformLabel(platform),
+            });
+        };
+
+        const getHeader = (i: number) => {
+            if (i === 0) {
+                return localize('Manage {{platform}} password', {
+                    platform: getCFDPlatformLabel(platform),
+                });
+            }
+            return localize('Manage password');
+        };
+
+        const onChangeActiveTabIndex = (i: number) => {
+            setIndex(i);
+        };
+
+        const steps = [
+            {
+                component: (
+                    <CFDPasswordManagerTabContent
+                        email={email}
+                        selected_login={selected_login}
+                        toggleModal={toggleModal}
+                        setPasswordType={setPasswordType}
+                        multi_step_ref={multi_step_ref}
+                        platform={platform}
+                        onChangeActiveTabIndex={onChangeActiveTabIndex}
+                        account_group={selected_account_group}
+                    />
+                ),
+            },
+            {
+                component: (
+                    <CFDPasswordReset
+                        server={selected_server}
+                        sendVerifyEmail={sendVerifyEmail as unknown as TSendVerifyEmail}
```

Comment:
Override the type here because need to confirm with Accounts Team if we are using the sendVerifyEmail in any other places. 
From `@deriv/api` the type is `()=> void` instead of `()=> Promise<VerifyEmailResponse>`